packages/node/src/evm/handlers/initialize-provider.ts

```diff
@@ -51,32 +50,10 @@ export async function initializeProvider(
   const state1 = state.refresh(initialState);
 
   // =================================================================
-  // STEP 2: Get current block number, and verify or set Airnode parameters
+  // STEP 2: Get current block number
   // =================================================================
-  const airnodeParametersFetchOptions = {
-    airnodeAdmin: state1.settings.airnodeAdmin,
-    airnodeRrpAddress: state1.contracts.AirnodeRrp,
-    authorizers: state1.settings.authorizers,
-    masterHDNode: state1.masterHDNode,
-    provider: state1.provider,
-  };
-  const [airnodeParametersLogs, airnodeParametersData] = await initialization.verifyOrSetAirnodeParameters(
-    airnodeParametersFetchOptions
-  );
-  logger.logPending(airnodeParametersLogs, baseLogOptions);
-
-  // If there is no Airnode parameters data, something has gone wrong
-  if (!airnodeParametersData) {
-    return null;
-  }
-
-  // If the Airnode parameters do not yet exist onchain then we can't start processing anything.
-  // This is to be expected for new Airnode deployments and is not an error case
-  if (airnodeParametersData.xpub === '') {
-    return state1;
-  }
-
-  const state2 = state.update(state1, { currentBlock: airnodeParametersData.blockNumber });
+  const currentBlock = await state1.provider.getBlockNumber();
```

Comment:
Apparently this is compatible with more chains (optimistic L2 stuff) than the way we did it before so this is an unexpected improvement

Reply:
wohooo! good point and I hadn't realized that 🎉